src/check_missing_alleles.py

```diff
@@ -0,0 +1,439 @@
+import csv
+import os
+from subprocess import Popen, PIPE
+
+
+def get_IMGT_data():
+    """Uses wget to retrieve the latest IMGT alleles + sequences from their github
+    These get deleted later after operations are complete
+    """
+    imgt_prot_URL = (
+        "https://raw.githubusercontent.com/ANHIG/IMGTHLA/Latest/hla_prot.fasta"
+    )
+    imgt_allele_URL = (
+        "https://raw.githubusercontent.com/ANHIG/IMGTHLA/Latest/Allelelist.txt"
+    )
+    process = Popen(["wget", imgt_prot_URL, imgt_allele_URL], stdout=PIPE, stderr=PIPE)
+    stdout, stderr = process.communicate()
+
+
+def update_chain_sequence():
+    """Updates the chain-sequence.tsv with missing alleles from IMGT
+
+    Returns:
+        A list of missing alleles ['A*02:01', 'B*02:01']
+    """
+
+    mro_alleles = set()
+    with open("../ontology/chain-sequence.tsv") as fh:
```

Comment:
This should also maybe be passed as an argument. If you don't run this from the `src/` directory, this file won't be found.